packages/fx-core/src/common/samples-config-v3.json

```diff
@@ -185,6 +185,16 @@
             "time": "5min to run",
             "configuration": "Ready for debug",
             "suggested": false
+        },
+        {
+          "id": "hello-world-teams-tab-and-outlook-add-in",
+          "title": "Hello World Teams tab and Outlook add-in",
+          "shortDescription": "A hello world project that contains both Teams tab and Outlook Add-in capability",
+          "fullDescription": "A hello world project that contains both Teams tab and Outlook Add-in capability",
```

Comment:
Does this need PM's assistant to add full description?

Reply:
@MuyangAmigo do we need a full description?

Reply:
Not really.